### PR TITLE
feat(data-testid): support for Callout, HighlightedCard, Hero, EmptyState and Headers

### PR DESCRIPTION
**Pull request by atabel**
WEB-2105

Figma specs:
https://www.figma.com/design/MopZWljO6aLvAoWfsmust0/testids?node-id=2861-8201&t=pvugflxmiHR0CJ43-4 (merged)
https://www.figma.com/design/H3OStb207o9VIJg5mE95S9/testids?node-id=2261-1732&t=FkQ42lukX47JRwf9-4 (merged)
https://www.figma.com/design/l750xJQse8DwuEmYFnXmAp/testids?node-id=3390-3251&t=aLSpLmHtjxBiqqhU-4 (merged)
https://www.figma.com/design/zUspRtleDhuq9f6BfLj7ZY/testids?node-id=5527-6701&t=mbnJHzFcUAd6KUZR-4 (merged)
https://www.figma.com/design/oaSYsDNA70XjZ3KOLdvp4T/testids-highlighed-card?node-id=9583-1016&t=xtM8CYBGQENVsd46-4 (in review)

### CODE REVIEW

File: src/__tests__/testid-test.tsx [hunk 1]
```diff
@@ -101,6 +109,15 @@ test('Cards test ids', () => {
     );
 });
 
+test('HighlightedCard test ids', () => {
+    checkTestIds(<HighlightedCard title="Title" description="Description" imageUrl="https://anyurl.com" />, [
```
Comment (yceballost) [inline src/__tests__/testid-test.tsx:113]: is the spec updated? I can't see "image" in specs
https://www.figma.com/design/tKdPOfcUALzVIh5oizFbm7/%F0%9F%94%B8-Cards-Specs?node-id=9583-1016&t=UQPGAgvhx5J6ErMZ-4

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/__tests__/testid-test.tsx (hunk 1 above)
Comment (atabel) [reply src/__tests__/testid-test.tsx:113]: nop, I didn't change the cards specs, I'll open a branch for this

Comment (atabel) [reply src/__tests__/testid-test.tsx:113]: https://www.figma.com/design/oaSYsDNA70XjZ3KOLdvp4T/testids-highlighed-card?node-id=9583-1016&t=xtM8CYBGQENVsd46-4

**Review by marcoskolodny — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Review by yceballost — APPROVED** [review]

**Pull request merged**